Backend-API/src/main/java/com/avangers/backendapi/repositories/UserRepository.java

```diff
@@ -0,0 +1,10 @@
+package com.avangers.backendapi.repositories;
+
+import com.avangers.backendapi.models.User;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface UserRepository extends JpaRepository<User, Long> {
+    boolean existsByEmail(String email);
+}
```

Comment:
```suggestion
@Repository
public interface UserRepository extends JpaRepository<User, Long> {
    boolean existsByEmail(String email);
    User findByEmail(String email); // this here
}
```

Reply:
@ReneVajda1 why do you want to add this function? we can, but we dont need it now right?

Reply:
because next two tasks will need it 

![image](https://github.com/gfa-cc-after/Avengers_PET_Clinic/assets/129347078/5c690392-a035-431b-84f5-6a44318d4369)

but dont mind adding it i just wanted to test out review and commenting and all

it doesnt have to be there now now now.

its up to you.